proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -206,6 +206,674 @@ function bufferNodeIs( test )
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check empty array';
+  var got = _.constructorIsBuffer( [] );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check string';
+  var got = _.constructorIsBuffer( 'string' );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check U8x';
+  var got = _.constructorIsBuffer( U8x );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check U8ClampedX';
+  var got = _.constructorIsBuffer( U8ClampedX );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check U16x';
+  var got = _.constructorIsBuffer( U16x );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check U32x';
+  var got = _.constructorIsBuffer( U32x );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check Ux';
+  var got = _.constructorIsBuffer( Ux );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check U64x';
+  var got = _.constructorIsBuffer( U64x );
+  var expected = true;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check I8x';
+  var got = _.constructorIsBuffer( I8x );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check I16x';
+  var got = _.constructorIsBuffer( I16x );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check I32x';
+  var got = _.constructorIsBuffer( I32x );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check Ix';
+  var got = _.constructorIsBuffer( Ix );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check I64x';
+  var got = _.constructorIsBuffer( I64x );
+  var expected = true;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check F32x';
+  var got = _.constructorIsBuffer( F32x );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check Fx';
+  var got = _.constructorIsBuffer( Fx );
+  var expected = true;
+  test.identical( got, expected );
+
+  test.case = 'check F64x';
+  var got = _.constructorIsBuffer( F64x );
+  var expected = true;
+  test.identical( got, expected );
+
+  /**/
+
+  if (Config.interpreter === 'njs')
+  {
+    test.case = 'check BufferNode';
+    var got = _.constructorIsBuffer( BufferNode );
+    var expected = false;
+    test.identical( got, expected );
+  }
+
+  /**/
+
+  test.case = 'check BufferRaw';
+  var got = _.constructorIsBuffer( BufferRaw );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check BufferRawShared';
+  var got = _.constructorIsBuffer( BufferRawShared );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check BufferView';
+  var got = _.constructorIsBuffer( BufferView );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  if( !Config.debug )
+    return;
```

Comment:
Next test cases are not throwing test cases. The `if` block is unnecessary condition. 

If you see unnecessary spaces, remove it please. ( Note, in this case this condition should be removed )

```js
if( !Config.debug )
return;
```